types/scene/wlr_scene.c

```diff
@@ -1872,23 +1884,18 @@ static void scene_entry_render(struct render_list_entry *entry, const struct ren
 				.use_optimized_blur = scene_rect->backdrop_blur_optimized,
 				.blur_data = &scene->blur_data,
 				.ignore_transparent = false,
+				.clipped_region = {
+					.area = rect_clipped_region_box,
+					.corner_radius = rect_clipped_region_corner_radius * data->scale,
+					.corners = rect_clipped_corners,
+				}
```

Comment:
Don't forget about adding clipped blur to the `scene_buffer` switch case as well

Reply:
Nvm, ignore this one